repository/src/main/java/com/epam/esm/config/DatabaseConfiguration.java

```diff
@@ -47,12 +53,42 @@ public DataSource customSqlData() {
     }
 
     @Bean
-    public PlatformTransactionManager transactionManager(DataSource dataSource) {
-        return new DataSourceTransactionManager(dataSource);
+    public JpaVendorAdapter jpaVendorAdapter() {
+        HibernateJpaVendorAdapter jpaVendorAdapter = new HibernateJpaVendorAdapter();
```

Comment:
You don`t need this configuration class in principal

Reply:
As I understood, after migrating to spring boot will delete it - https://github.com/egor-geller/certificate/pull/3#discussion_r739015908